ipaserver/install/cainstance.py

```diff
@@ -400,7 +400,8 @@ def configure_instance(self, host_name, dm_password, admin_password,
                     self.step("Importing RA key", self.__import_ra_key)
                 else:
                     self.step("importing RA certificate from PKCS #12 file",
-                              lambda: self.import_ra_cert(ra_p12))
+                              lambda: self.import_ra_cert(ra_p12,
+                                                          self.dm_password))
```

Comment:
Could you please put this into a `__import_ra_cert` helper method so we get rid of this ugly lambda expression?